Resources/MessageQueue/MQConnectionManager.py

```diff
@@ -0,0 +1,387 @@
+
+      if not _removeMessanger(cStorage = self._connectionStorage, mqConnection = conn, destination = dest, messangerId = messangerId):
+        return S_ERROR('Failed to stop the connection!The messanger:'+ messangerId + ' does not exists!')
+      else:
+        if 'consumer' in messangerId:
+          result = self.unsubscribe(connector, destination = dest, messangerId = messangerId)
+          if not result['OK']:
+            return result
+      if not _connectionExists(self._connectionStorage, conn):
+        return self.disconnect(connector)
+      return S_OK()
+    finally:
+      self.lock.release()
+
+  def getAllMessangers(self):
+    """ Function returns a list of all messangers registered in connection storage.
+    Returns:
+      S_OK/S_ERROR: with the list of strings in the pseudo-path format e.g.
+            ['blabla.cern.ch/queue/test1/consumer1','blabal.cern.ch/topic/test2/producer2']
+    """
+    self.lock.acquire()
+    try:
+      return S_OK(_getAllMessangersInfo(cStorage = self._connectionStorage))
+    finally:
+      self.lock.release()
+
+  def removeAllConnections(self):
+    """ Function removes all existing connections and calls the disconnect
+        for connectors.
+    Returns:
+      S_OK/S_ERROR:
+    """
+    self.lock.acquire()
+    try:
+      connections = _getAllConnections(cStorage = self._connectionStorage)
+      for conn in connections:
+        connector =_getConnector(self._connectionStorage, conn)
+        if connector:
+          self.disconnect(connector)
+      self._connectionStorage = {}
+      return S_OK()
+    finally:
+      self.lock.release()
+
+# Set of 'private' helper functions to access and modify the message queue connection storage.
+
+def _getConnection(cStorage, mqConnection):
+  """ Function returns message queue connection from the storage.
+  Args:
+    cStorage(dict): message queue connection storage.
+    mqConnection(str): message queue connection name.
+  Returns:
+    dict:
+  """
+  return cStorage.get(mqConnection, {})
+
+def _getAllConnections(cStorage):
+  """ Function returns a list of all connection names in the storage
+  Args:
+    cStorage(dict): message queue connection storage.
+  Returns:
+    list:
+  """
+  return cStorage.keys()
+
+def _getConnector(cStorage, mqConnection):
+  """ Function returns MQConnector from the storage.
+  Args:
+    cStorage(dict): message queue connection storage.
+    mqConnection(str): message queue connection name.
+  Returns:
+    MQConnector or None
+  """
+  return _getConnection(cStorage, mqConnection).get("MQConnector", None)
+
+def _setConnector(cStorage, mqConnection, connector):
+  """ Function returns MQConnector from the storage.
+  Args:
+    cStorage(dict): message queue connection storage.
+    mqConnection(str): message queue connection name.
+    connector(MQConnector):
+  Returns:
+    bool: False if connection does not exit
+  """
+  if not _getConnection(cStorage, mqConnection):
```

Comment:
I would do:

  connDict = _getConnection( cStorage, mqConnection )
  if not connDict:
    return False
  connDict["MQConnector"] = connector
  return True

If even one line more, it is more explicit. The more readable is the code the better it is for all the developers (including you).